src/Symfony/Component/HttpFoundation/Tests/Fixtures/send_headers_cookie_samesite.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Response;
+
+require __DIR__ . '/common.inc';
+
+$r = new Response();
+$r->headers->set('Date', 'Thu, 02 Feb 2012 12:12:12 GMT');
+$r->headers->setCookie(new Cookie('SameSite', 'Strict'));
```

Comment:
this creates a cookie with name SameSite, and value Strict, that's not the feature we want to test
instead, we want to enable the $sameSite parameter of the constructor of the Cookie class.

Reply:
Done, thanks.